### DISCUSSION TITLE
[passport-google-oauth20] email_verified is incorrect typed

### DISCUSSION THREAD

**Discussion by DimitarNikolaevVasilev** [DefinitelyTyped/DefinitelyTyped#64769] [Issues with a @types package]
The type for email_verified currently is "true" or "false" but after trying and debuging the library the type is boolean.
So i was doing user.emails![0].verified === 'true' to convert to boolean which was wrong.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/passport-google-oauth20/index.d.ts#L63

[a comment by an automated account is not shown]

**Comment by ezintz** [comment]
Do you have any example to try? I just checked the documentation and implementation and it looks like a string value is still correct. Is there any chance you are using a custom parser for JSON?

References:
https://developers.google.com/identity/openid-connect/openid-connect#obtainuserinfo
https://github.com/jaredhanson/passport-google-oauth2/blob/master/lib/profile/openid.js
